src/components/Map.js

```diff
@@ -177,7 +179,10 @@ class Map extends Component {
         el.className = 'marker';
         el.id = feature.properties.id;
         el.style.backgroundImage = `url(${feature.properties.thumbnail}), url(${placeholderImage}) `;
-        el.addEventListener('click', () => this.setState({openDialog:true,feature}));
+        el.addEventListener('click', () => {
+          reactGAevent('User','Photo in the map');
```

Comment:
how is "Photo in the map" part of the "User" category ?

Reply:
changed